.travis.yml

```diff
@@ -1,42 +1,47 @@
 cache: cargo
 before_cache:
-  - find ./target/debug -type f -maxdepth 1 -delete
-  - rm -fr ./target/debug/{deps,.fingerprint}/{*ra_*,*test*,*tools*,*gen_lsp*,*thread_worker*}
-  - rm -f  ./target/.rustc_info.json
+    - find ./target/debug -type f -maxdepth 1 -delete
+    - rm -fr ./target/debug/{deps,.fingerprint}/{*ra_*,*test*,*tools*,*gen_lsp*,*thread_worker*}
+    - rm -f  ./target/.rustc_info.json
 
 env:
-  - CARGO_INCREMENTAL=0
+    - CARGO_INCREMENTAL=0
 
 build: &rust_build
-  language: rust
-  rust: 1.31.1
-  script:
-    - cargo gen-tests --verify
-    - cargo gen-syntax --verify
-    - cargo test
+    language: rust
+    rust: 1.31.1
+    script:
+        - cargo gen-tests --verify
+        - cargo gen-syntax --verify
+        - RUSTFLAGS='-D warnings' cargo test
```

Comment:
Why do we set `-D warnings` both here and via `env`? I'd prefer to just use `env`, but I was unable to figure out the proper yaml syntax for that :-)

Reply:
I forgot to remove it :). Will fix now